OPERATING.md

```diff
@@ -0,0 +1,114 @@
+# Operating
+
+How to operate the GP2GP adaptor
+
+## Adaptor Process
+
+TODO: Sequence diagram "user journey" of Spine messages, tasks, and GPCC requests
+
+## Logging and Tracing
+
+```
+yyyy-MM-dd HH:mm:ss.SSS Level=INFO Logger=u.n.a.g.e.m.AgentDirectoryExtractor ConversationId=18262a7b-cb4c-4aef-8dd8-7ffc3dda67aa TaskId=ad4e53cc-9b4e-4e07-8937-f0f91887b21d Thread=java-thread-1 Message="The log message"
+```
+
+* Level: The logging level DEBUG/INFO/WARN/ERROR
+* Logger: The name of the Java class that emitted the message
+* ConversationId: The identifier correlating all GP2GP message for a single patient transfer
+* TaskId: The unique identifier of the task
+* Message: The log message
+
+## Object Storage (AWS S3 / Azure Blob)
+
+TODO: Document storage scheme
+
+## AMQP Message Broker Requirements
+
+* The broker must be configured with a limited number of retries and dead-letter queues
+* It is the responsibility of the GP supplier to configure adequate monitoring against the dead-letter queues that allows ALL undeliverable messages to be investigated fully.
+* The broker must use persistent queues to avoid loss of data
+
+**Using AmazonMQ**
+
+* A persistent broker (not in-memory) must be used to avoid data loss.
+* A configuration profile that includes settings for [retry and dead-lettering](https://activemq.apache.org/message-redelivery-and-dlq-handling.html) must be applied.
+* AmazonMQ uses the scheme `amqp+ssl://` but this **MUST** be changed to `amqps://` when configuring the adaptor.
+
+**Using Azure Service Bus**
+
+* The ASB must use [MaxDeliveryCount and dead-lettering](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues#exceeding-maxdeliverycount)
+* Azure Service Bus may require some parameters as part of the URL configuration. For example: `GP2GP_AMQP_BROKERS=amqps://<NAME>.servicebus.windows.net/;SharedAccessKeyName=<KEY NAME>;SharedAccessKey=<KEY VALUE>`
+
```

Comment:
These are the same instructions used for the NHAIS adaptor so I'd be inclined to keep it the same. It may be that there are multiple ways to specify the configuration as is the case for mongo.